pkg/services/ngalert/image/service.go

```diff
@@ -9,36 +9,40 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/grafana/grafana/pkg/components/imguploader"
+	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/services/dashboards"
-	"github.com/grafana/grafana/pkg/services/ngalert/models"
+	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
 	"github.com/grafana/grafana/pkg/services/ngalert/store"
 	"github.com/grafana/grafana/pkg/services/rendering"
 	"github.com/grafana/grafana/pkg/services/screenshot"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
-//go:generate mockgen -destination=mock.go -package=image github.com/grafana/grafana/pkg/services/ngalert/image ImageService
-type ImageService interface {
-	// NewImage returns a new image for the alert instance.
-	NewImage(ctx context.Context, r *models.AlertRule) (*models.Image, error)
-}
+const (
+	screenshotTimeout  = 10 * time.Second
+	screenshotCacheTTL = 60 * time.Second
```

Comment:
maybe it's time to make it configurable?

Reply:
I agree, follow up PR?